data/transactions/logic/opcodes.go

```diff
@@ -493,8 +493,9 @@ var OpSpecs = []OpSpec{
 	{0x59, "extract_uint16", opExtract16Bits, proto("bi:i"), 5, opDefault()},
 	{0x5a, "extract_uint32", opExtract32Bits, proto("bi:i"), 5, opDefault()},
 	{0x5b, "extract_uint64", opExtract64Bits, proto("bi:i"), 5, opDefault()},
-	{0x5c, "base64_decode", opBase64Decode, proto("b:b"), fidoVersion, field("e", &Base64Encodings).costByLength(1, 1, 16, 0)},
-	{0x5d, "json_ref", opJSONRef, proto("bb:a"), fidoVersion, field("r", &JSONRefTypes).costByLength(25, 2, 7, 1)},
+
+	{0x5e, "base64_decode", opBase64Decode, proto("b:b"), fidoVersion, field("e", &Base64Encodings).costByLength(1, 1, 16, 0)},
```

Comment:
@jannotti For my understanding and to confirm it's deliberate:  Did you stagger `base64_decode` and `json_ref` opcodes in order to slot in `extract` opcodes?

Reply:
Yes, since they weren't released yet, we had free reign to relocate for better organization.

Reply:
@jannotti Got it - resolving.